tOpt/coordinates_batch.py

```diff
@@ -26,6 +26,9 @@ def __init__(self, allowed_atom_num:Sequence[int], dtype ):
         self.allowed_atom_num = frozenset(allowed_atom_num)
         self.dtype           = dtype
 
+        self.atom_indices    = None  # tensor nconf,nAtoms  with atomic indices
+        self._at_index_list  = []
```

Comment:
This will make the changes not compatible with other networks. Could you move all code involving at_index into the  ANI1xNet and ANI2xNet classes?